docs/ToggleBadge.md

```diff
@@ -15,7 +15,7 @@ Use the `<ToggleBadge />` component to render a primitive badge.
 
 Prop | Type | Description
 ---|---|---
-selected | boolean | gives different background color and makes font bold
```

Comment:
So it seems like the font weight in `ToggleBadge` is still using `props.theme.bold}` in the new code. Would it be correct to update the doc to remove the emphasis of making the font bold.